src/parser/args.rs

```diff
@@ -237,12 +174,14 @@ The type of it is an array of BlockDeviceConfigInfo, e.g.
         display_order = 2
     )]
     pub fs: String,
-}
 
-/// Config boot source including rootfs file path
-#[derive(Args, Debug, Deserialize, Serialize, Clone)]
-#[clap(arg_required_else_help = true)]
-pub struct BootArgs {
+    // feature for log
+    #[clap(long, value_parser, default_value = "dbs-cli.log", display_order = 1)]
+    pub log_file: String,
+
+    #[clap(long, value_parser, default_value = "Info", display_order = 1)]
```

Comment:
Is "Debug" a better default value? The reason is that dbs-cli is a debug tool for Dragonball.

Reply:
changed.